usr/share/rear/conf/GNU/Linux.conf

```diff
@@ -18,6 +18,7 @@ sfdisk
 
 # progs to take along
 PROGS+=(
+fuser
```

Comment:
When 'fuser' is only in PROGS but not in REQUIRED_PROGS
I see now that my code in umount_mountpoint_retry_lazy()
```
fuser ... || Log "Presumably 'fuser' does not support the -M option"
```
shows a wrong log message when 'fuser' does not exist
so it should be better something like
```
fuser ... || Log "'fuser' failed (presumably it may not support the -M option)"
```
I am now also wondering why I did
```
fuser -v -M -m "$mountpoint" 1>&2 || Log ...
```
that stdout redirection to stderr because this
should be no longer needed according to
https://github.com/rear/rear/wiki/Coding-Style#what-to-do-with-stdin-stdout-and-stderr